src/components/table/table-row/table-row.component.js

```diff
@@ -23,6 +23,7 @@ import ActionPopover from '../../action-popover';
  */
 class TableRow extends React.Component {
   state = {
+    rowNode: null,
```

Comment:
Does this need comments to annotate the documentation?

Reply:
Yes, for consistency with the other `state` variables in this component (`highlighted` and `selected`). I'll fix this.

Reply:
Fixed in 42fda8d :thumbsup: